IDEAS/Buildings/Components/InternalGains/Occupants.mo

```diff
@@ -8,7 +8,7 @@ model Occupants "Constant sensible, latent and CO2 heat production per person"
   parameter Real radFra(min=0,max=1) = occupancyType.radFra
     "Radiant fraction of sensible heat exchange, default based on Ashrae fundamentals chap 18.4 for low air velocity";
 protected
-  constant Modelica.SIunits.SpecificEnthalpy lambdaWater = 2418000
+  constant Modelica.SIunits.SpecificEnthalpy lambdaWater = if (Medium.nC) >= 1 then Medium.enthalpyOfCondensingGas(T=273.15+35) else 2566120
```

Comment:
Why `Medium.nC`? Did you intend to use `Medium.nX` or `Medium.nXi`?

Reply:
I may have copy pasted from the lines below